schemas/iso19110/src/main/plugin/iso19110/loc/eng/labels.xml

```diff
@@ -261,22 +264,102 @@
     <label>Scope</label>
     <description>Scope definition</description>
   </element>
-  <element name="gmx:versionNumber">
-    <label>Version</label>
-    <description>Catalogue version</description>
-  </element>
-  <element name="gmx:versionDate">
-    <label>Date</label>
-    <description>Catalogue date</description>
-  </element>
-  <element name="gmx:fieldOfApplication">
-    <label>Field of application</label>
-    <description>Field of application</description>
+
+  <element name="gmx:subCatalogue">
+    <label>Sub catalogue</label>
+    <btnLabel>Add sub catalogue</btnLabel>
+    <description>Describes a sub catalogue</description>
   </element>
+
   <element name="gco:nilReason">
     <label>Nil reason</label>
-    <description/>
-    <help/>
-    <condition/>
   </element>
+
```

Comment:
Not sure this is necessary (except for gfc namepace), 19110 is supposed to delegate labels to ISO19139 for its namespaces eg. https://github.com/geonetwork/core-geonetwork/blob/master/schemas/iso19110/src/main/plugin/iso19110/layout/layout.xsl#L78-L83

Reply:
Indent is wrong - 2 spaces needed

Reply:
mmm, i got warnings in log about missing translations, so maybe we have to look at the delegation?

Reply:
The warnings still happen, but translations are indeed delegated to iso19139